app/controllers/idv/personal_key_controller.rb

```diff
@@ -19,7 +19,10 @@ def show
     def update
       user_session[:need_personal_key_confirmation] = false
 
-      analytics.idv_personal_key_submitted(address_verification_method: address_verification_method)
+      analytics.idv_personal_key_submitted(
```

Comment:
If you could, it's also helpful to update the documentation for the analytics events: https://github.com/18F/identity-idp/blob/4636859fb4ba0e94e99b3c58d0d317688ff08985/app/services/analytics_events.rb#L1080

(if possible, same for `idv_review_complete` and `idv_final`)